csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -26,11 +26,12 @@
     - Visual Studio.
     -->
   <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>netstandard1.1</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="SourceLink.Create.CommandLine" PrivateAssets="All" Version="2.7.6"/>
+    <PackageReference Include="SourceLink.Create.CommandLine" PrivateAssets="All" Version="2.7.6" />
+    <PackageReference Include="System.Memory" Version="4.5.2" />
```

Comment:
Adding a new dependency is also usually considered a "breaking change". Maybe make this conditional on the target framework?

Reply:
> Adding a new dependency is also usually considered a "breaking change". Maybe make this conditional on the target framework?

Adding a new dependency isn't a breaking change. Removing one is though. I think we should potentially bump the minor version here.

Reply:
Minor version will be bumped because this change won't be available before protobuf v3.8.x is out.